### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader:Narrator is not reading the name of the links "FolderIcon" and "Slash icon" is deascriptive manner.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3986]
EnvironmentDetails:
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 Steps to reproduce:
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the "folder" icon.

**Actual:**
Narrator is not reading the name of the links "FolderIcon" and "Slash icon" is deascriptive manner. #3986 **Expected:**
There should be proper description about the link. May  there could be a descriptive tooltip for the link.
**UserImpact:**
User will not get know the name and any extra description about the link or what it does.
**Attachment:**
![bugattachment](https://user-images.githubusercontent.com/39204909/45806321-5fc04d80-bcde-11e8-8ff0-ce3832d7a765.png)

[Presentation2.pptx](https://github.com/jupyter/notebook/files/2400583/Presentation2.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
